### PR TITLE
Separate network elements info types second part

### PR DESCRIPTION
**Pull request by AAJELLAL**

### CODE REVIEW

File: src/utils/rest-api.js [hunk 1]
```diff
@@ -486,12 +486,13 @@ export function fetchTwoWindingsTransformers(
...
-    return fetchEquipments(
+    return fetchNetworkElementsInfos(
         studyUuid,
         currentNodeUuid,
         substationsIds,
-        'Two windings transformers',
-        '2-windings-transformers'
+        EQUIPMENT_TYPES.TWO_WINDINGS_TRANSFORMER.type,
+        EQUIPMENT_INFOS_TYPES.TAB.type,
+        true
```
Comment (archendel) [inline src/utils/rest-api.js:495]: why do all "inUpstreamBuiltParentNode" are now passed as "true" ? They were passed as "false" before if I'm not wrong

File: src/utils/rest-api.js [hunk 2]
```diff
@@ -486,12 +486,13 @@ export function fetchTwoWindingsTransformers(
     currentNodeUuid,
     substationsIds
 ) {
-    return fetchEquipments(
+    return fetchNetworkElementsInfos(
```
Comment (archendel) [inline src/utils/rest-api.js:489]: I think we could improve readability by changing all "fetch{equipmen}" to "fetch{equipment}{infoType}" or by passing "infoType" as argument

Here we don't really know which info type we are fetching without taking a look at the function implementation

**Review by archendel — COMMENTED** [review]
some comments, will test soon

**Review by SlimaneAmar — APPROVED** [review]

File: src/utils/rest-api.js (hunk 2 above)
Comment (archendel) [reply src/utils/rest-api.js:489]: yes but for all fetchGenerators, fetchBatteries, ... we don't know which infoType we are fetching

**Review by archendel — APPROVED** [review]

**Pull request merged**